install/etcd/install.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: etcd
+parameters:
+- name: NAMESPACE
+  value: kube-system
+objects:
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: etcd
+    namespace: ${NAMESPACE}
+  spec:
+    selector:
+      openshift.io/component: etcd
+    ports:
+    - name: listen
+      port: 4001
```

Comment:
2379 ?

Reply:
and reason not to switch? 4001 is like 3+ years old.

Reply:
> and reason not to switch? 4001 is like 3+ years old.

doesn't currently work that way?